en/guides/manage-devices/index.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Introduction
+table_of_contents: true
+---
+
+# Introduction
+
+On some systems it is appropriate to have a system user. For example, with a system user, one can ssh to the device and run commands.
+
+By default on first boot, stock Core images run a program called console-conf. If the user has a keyboard and monitor attached to the device, they can use console-conf to configure the network and add a user account. This user account can then be used to ssh to the device for various management purposes.
```

Comment:
@tony, surprisingly, the user created by console conf is not a system user in the following sense:
knitzsche@localhost:~$ snap known system-user
knitzsche@localhost:~$ snap managed
true

I'll clarify the text on that point. 

On the other point about users created by console-conf, yes, I'll add that.